### PR TITLE
LPD-35399 Extract locators and functions from ModelBuilderPage to other new pages according to context

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-35399

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:playwright

**Comment by rebsilva** [comment]
just started reviewing! :cherry_blossom:

**Review by rebsilva — CHANGES_REQUESTED** [review]
hi @danvb112! great work on this, just made some suggestions so please let me know what you think!

File: modules/test/playwright/pages/object-web/model-builder/ModelBuilderRightSidebarPage.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	readonly modalDeleteObjectRelationshipConfirmationButton: Locator;
+	readonly rightSidebar: Locator;
+	readonly rightSidebarObjectDefinitionActivateObject: Locator;
+	readonly rightSidebarObjectDefinitionEntryTitleField: Locator;
+	readonly rightSidebarObjectDefinitionLabel: Locator;
+	readonly rightSidebarObjectDefinitionLabelLocalizationButton: Locator;
+	readonly rightSidebarObjectDefinitionPanelLink: Locator;
+	readonly rightSidebarObjectDefinitionPluralLabel: Locator;
+	readonly rightSidebarObjectDefinitionPluralLabelLocalizationButton: Locator;
+	readonly rightSidebarObjectDefinitionScope: Locator;
```
[the remark this answers is not in the document]
Comment (danvb112) [reply modules/test/playwright/pages/object-web/model-builder/ModelBuilderRightSidebarPage.ts:25]: Agree @rebsilva, I've removed all the `rightSidebar` suffixes from locators.

File: modules/test/playwright/pages/object-web/model-builder/ModelBuilderRightSidebarPage.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		this.modelBuilderLeftSidebarPage = new ModelBuilderLeftSidebarPage(
+			page
+		);
+		this.modalDeleteObjectRelationshipTextField = page.getByPlaceholder(
+			'Confirm Relationship Name'
+		);
+		this.modalDeleteObjectRelationshipConfirmationButton = page.getByRole(
+			'button',
+			{exact: true, name: 'Delete'}
+		);
```
[the remark this answers is not in the document]
Comment (danvb112) [reply modules/test/playwright/pages/object-web/model-builder/ModelBuilderRightSidebarPage.ts:47]: Totally agree @rebsilva, I removed the `modalDeleteObjectRelationshipConfirmationButton` locator to use only the `deleteButton`, great catch!

File: modules/test/playwright/pages/object-web/model-builder/ModelBuilderObjectDefinitionNodePage.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+export class ModelBuilderObjectDefinitionNodePage {
+	readonly addObjectFieldButton: Locator;
+	readonly deleteObjectDefinitionOption: Locator;
+	readonly modelBuilderDiagramPage: ModelBuilderDiagramPage;
+	readonly newObjectFieldSelectPicklist: Locator;
+	readonly newObjectFieldSaveButton: Locator;
+	readonly newObjectRelationshipLabel: Locator;
+	readonly newObjectRelationshipTitle: Locator;
+	readonly newObjectRelationshipType: Locator;
```
[the remark this answers is not in the document]
Comment (danvb112) [reply modules/test/playwright/pages/object-web/model-builder/ModelBuilderObjectDefinitionNodePage.ts:19]: Yeah I agree, I've changed all the relationship locator names based on your suggestion! Thanks @rebsilva !

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:playwright

**Comment by danvb112** [comment]
ci:test:playwright

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
Sent to: https://github.com/liferay-objects/liferay-portal/pull/3638

**Pull request closed without merging**